slices/node-undici.yaml

```diff
@@ -0,0 +1,13 @@
+package: node-undici
+
+slices:
+  libs:
+    essential:
+      - node-busboy_libs
+    contents:
```

Comment:
you've kept some files out (like `/usr/share/nodejs/undici/index.d.ts`, `/usr/share/nodejs/undici/types/*`). If that was the intention, could you please add a comment explaining the rationale?

Reply:
P.s. I noticed that most of these left-out files are TypeScript, but at the same time, both in this slice and other lib slices above, .ts files are often included alongside.

Reply:
Node.js can not run TypeScript files. First, they must be compiled into JavaScript files, which will strip all the type information. So typing data is not required during the runtime.

Reply:
Ok thanks. So then can you pls either remove all `.ts` files from slices, or add a comment to the slices have have or don't have them, explaining the rationale?

Reply:
I will remove all the `.ts` files.

Reply:
ok thanks. feel free to resolve this once you've committed the changes ;)

Reply:
All TypeScript stubs removed.